### PR TITLE
TAC

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

File: content/Timer_and_Divider_Registers.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ Bits 1-0 - Input Clock Select
 ```
 
 ::: tip NOTE
+Select the clock frequency first, then enable the Timer.
```
Comment (ISSOtm) [inline content/Timer_and_Divider_Registers.md:43]: Do you mean that it should be done by a separate write? If so, why should that be done?

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: content/Timer_and_Divider_Registers.md (hunk 1 above)
Comment (AlvaroBurnett) [reply content/Timer_and_Divider_Registers.md]: Explained

File: content/Timer_and_Divider_Registers.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ Bits 1-0 - Input Clock Select
 ```
 
 ::: tip NOTE
+Select the clock frequency first, then enable the Timer. Starting the Timer before
+or at the same time as the clock frequency is selected may result in excessive count up operation.
```
Comment (ISSOtm) [inline content/Timer_and_Divider_Registers.md:44]: That sounds copy-pasted from the official manual; is it? If so, then research as to what the actual behavior is may be needed. (Looking to eliminate that "may".)

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: content/Timer_and_Divider_Registers.md (hunk 2 above)
Comment (AlvaroBurnett) [reply content/Timer_and_Divider_Registers.md:44]: We could ask @Gekkio, if he shows up in a near future. In the meantime, following the recommendation doesn't hurt.

Comment (ISSOtm) [reply content/Timer_and_Divider_Registers.md:44]: There's [already a document detailing the circuits](https://gbdev.io/pandocs/#timer-obscure-behaviour), so we should be able to figure this out without bothering him ^^

Comment (avivace) [reply content/Timer_and_Divider_Registers.md:44]: I agree. No point in reporting an apparently non-deterministic behaviour if we don't have enough proof

**Review by avivace — CHANGES_REQUESTED** [review]

**Pull request closed without merging**